source/apiVolontaria/apiVolontaria/tests/tests_view_ResetPassword.py

```diff
@@ -239,19 +241,18 @@ def test_create_new_token_without_email_service(self):
         Ensure we can have a new token to change our password
         """
         data = {
-            'username': self.user.username,
+            'email': self.user.email
         }
 
-        # Test that one message was sent:
-        self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(mail.outbox[0].to, [self.user.email])
-
         response = self.client.post(
             reverse('reset_password'),
             data,
             format='json',
         )
 
+        # Test that one message was sent:
+        self.assertEqual(len(mail.outbox), 0)
```

Comment:
Ceci teste qu'aucun message n'ai été envoyé avec succès, ce qui est le contraire du commentaire.

Reply:
Idem, marked as resolved but not resolved.

Reply:
Je ne suis pas certain de ce que tu dis @francisjeanneau, ce code est ici pour montrer que l'email échoue, c'est le code actuellement en place qui est faux. Le token n'est pas créé, on ne veux pas envoyer de courriel